schema/common.gql

```diff
@@ -469,6 +469,33 @@ type ConfigMap implements Node & KubernetesResource {
   events: EventConnection! @goField(forceResolver: true)
 }
 
+"`ObjectReference` contains enough information to let you inspect or modify the referred object."
+type ObjectReference {
+  "Kind of the referent."
+  kind: String
+
+  "Namespace of the referent."
+  namespace: String
+
+  "Name of the referent."
+  name: String
+
+  "UID of the referent."
+  uid: String
+
+  "API version of the referent."
+  resourceVersion: String
```

Comment:
I should have been more specific in my last comment - I think we should omit `uid` and `resourceVersion` as well as `fieldPath`.

Reply:
On it.